### PR TITLE
test: AES module E2E tests

### PR DESCRIPTION
**Pull request by gantunesr**
## **Description**

This PR adds a new UI exclusive to test the specific methods from the `Encryptor` class and the underlying AES native module.

## **Related issues**

Relates: https://github.com/MetaMask/mobile-planning/issues/1551

## **Manual testing steps**

No manual QA required

## **Screenshots/Recordings**

https://github.com/MetaMask/metamask-mobile/assets/17601467/c7e7bde6-8f68-4645-b4e4-ff86d8259357

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: app/components/Nav/Main/MainNavigator.js [hunk 1]
```diff
@@ -273,6 +274,11 @@ const SettingsFlow = () => (
       name={Routes.RAMP.ACTIVATION_KEY_FORM}
       component={RampActivationKeyForm}
     />
+    <Stack.Screen
+      name="AesCryptoTestForm"
+      component={AesCryptoTestForm}
+      options={AesCryptoTestForm.navigationOptions}
+    />
```
Comment (gantunesr) [inline app/components/Nav/Main/MainNavigator.js:281]: Add `isTest` conditional

File: app/components/Views/AesCryptoTestForm/Form.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { View, SafeAreaView, Keyboard, TextInput } from 'react-native';
+
+import Text, {
+  TextVariant,
+} from '../../../component-library/components/Texts/Text';
+// import TextField, {
+//   TextFieldSize,
+// } from '../../../component-library/components/Form/TextField';
```
Comment (gantunesr) [inline app/components/Views/AesCryptoTestForm/Form.tsx:9]: Remove

File: app/components/Views/AesCryptoTestForm/styles.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { StyleSheet } from 'react-native';
+
+import Device from '../../../util/device';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (gantunesr) [inline app/components/Views/AesCryptoTestForm/styles.ts:5]: Remove comment

**Review by cortisiko — APPROVED** [review]
Looks good Guto! 🌮 🌮 🌮

**Review by owencraston — APPROVED** [review]

**Review by Cal-L — APPROVED** [review]
Real nice. LGTM

**Pull request merged**